app/models/order.rb

```diff
@@ -2,8 +2,11 @@
 
 class Order < ApplicationRecord
   has_many :line_items, dependent: :destroy
-
+  belongs_to :user, optional: true
   validates :name, :address, :email, :pay_type, presence: true
+  scope :active, -> { where(mark: 'active') }
+  scope :completed, -> { where(mark: 'completed') }
```

Comment:
not sure but please check. I suppose that these scopes were created by the enum. Please try to remove that and call Order.active